db/migrations/01_create_peeps_table.sql

```diff
@@ -0,0 +1 @@
+CREATE TABLE peeps(id SERIAL PRIMARY KEY, message VARCHAR(60), time TIMESTAMP WITH TIME ZONE NOT NULL);
```

Comment:
For 'message' I used VARCHAR(255) with my theory that it was closer to the character limit in a Twitter tweet, no idea if this is true or not